proto/lbm/foundation/v1/foundation.proto

```diff
@@ -27,7 +27,7 @@ message Member {
   string metadata = 2;
 
   // added_at is a timestamp specifying when a member was added.
-  google.protobuf.Timestamp added_at = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  google.protobuf.Timestamp added_at = 3 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
```

Comment:
I think this is breaking change. 
Is this change required?

Reply:
OK, I will revert it.